pkg/cloud/services/s3/s3_test.go

```diff
@@ -694,11 +694,9 @@ func TestDeleteObject(t *testing.T) {
 		}
 	})
 
-	t.Run("succeeds_when_bucket_has_already_been_removed", func(t *testing.T) {
+	t.Run("succeeds_when", func(t *testing.T) {
```

Comment:
any reason to change test case name?

Reply:
Yes because I split it in 3 subcases:
```
     --- PASS: TestDeleteObject/succeeds_when (0.00s)
        --- PASS: TestDeleteObject/succeeds_when/bucket_has_already_been_removed (0.00s)
        --- PASS: TestDeleteObject/succeeds_when/object_has_already_been_removed (0.00s)
        --- PASS: TestDeleteObject/succeeds_when/bucket_or_object_not_found (0.00s) 
```